lib/challenges/lec1.py

```diff
@@ -59,4 +64,4 @@ def checkResult(self):
 
     def lec12_solution(self, x, y):
```

Comment:
We shouldn't have the actual solution in the Challenge class. Instead, we can use a list of input and expected output and compare them with the actual output.